integration_tests/test_emscoring.py

```diff
@@ -0,0 +1,67 @@
+import tempfile
+from pathlib import Path
+
+import pytest
+import shutil
+import pandas as pd
+
+from haddock.modules.scoring.emscoring import HaddockModule as EmscoringModule
+from haddock.modules.scoring.emscoring import DEFAULT_CONFIG as DEFAULT_EMSCORING_CONFIG
+from haddock.libs.libontology import PDBFile, TopologyFile
+from integration_tests.test_alascan import MockPreviousIO
```

Comment:
Best to not import the mocked method from another test. I see you also define it below, so maybe this is an old import?